pkg/asset/agent/installconfig.go

```diff
@@ -330,38 +400,6 @@ func warnUnusedConfig(installConfig *types.InstallConfig) {
 	case vsphere.Name:
 		vspherePlatform := installConfig.Platform.VSphere
 
-		if vspherePlatform.DeprecatedVCenter != "" {
```

Comment:
Should the warnings about "Deprecated" fields be kept since these fields still really are ignored?

Reply:
They're not ignored and can still be used. They are transformed into the new vcenters/failuredomains format by the installer.